polaris-react/loom.config.ts

```diff
@@ -74,6 +77,21 @@ function jestAdjustmentsPlugin() {
   });
 }
 
+function stylelintAdjustmentsPlugin() {
+  return createWorkspacePlugin('Polaris.Stylelint', ({tasks: {lint}}) => {
+    lint.hook(({hooks}) => {
+      hooks.configure.hook((configure) => {
+        // Modify the maximum number of allowed warnings from the default of 0
+        configure.stylelintFlags?.hook((flags) => ({
+          ...flags,
+          // Using -1 to allow all warnings
+          maxWarnings: -1,
```

Comment:
Yes!

[![Yarn lint output](https://screenshot.click/04-09-vnep8-fgmf6.png)](https://screenshot.click/04-09-vnep8-fgmf6.png)

Reply:
On the up side this works, on the down side it seems loom swallows output of the warnings (contrast this with running `yarn run stylelint '**/*.scss' --max-warnings=9999` manually)

Reply:
The root-level linting looks good to me! Was also wanting to make this improvement as well.

Not using loom sounds good in this case. I also agree not swallowing the warnings is ideal 👍 Question, would it be better no just omit the `--max-warnings` flag instead of using `9999`?